1.16.5/src/main/java/io/github/axolotlclient/modules/hypixel/bedwars/BedwarsMod.java

```diff
@@ -0,0 +1,230 @@
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * For more information, see the LICENSE file.
+ */
+
+package io.github.axolotlclient.modules.hypixel.bedwars;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import io.github.axolotlclient.AxolotlClientConfig.options.BooleanOption;
+import io.github.axolotlclient.AxolotlClientConfig.options.EnumOption;
+import io.github.axolotlclient.AxolotlClientConfig.options.OptionCategory;
+import io.github.axolotlclient.modules.hypixel.AbstractHypixelMod;
+import io.github.axolotlclient.util.events.Events;
+import io.github.axolotlclient.util.events.impl.ReceiveChatMessageEvent;
+import io.github.axolotlclient.util.events.impl.ScoreboardRenderEvent;
+import io.github.axolotlclient.util.events.impl.WorldLoadEvent;
+import lombok.Getter;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.PlayerListEntry;
+import net.minecraft.scoreboard.Scoreboard;
+import net.minecraft.scoreboard.ScoreboardPlayerScore;
+import net.minecraft.scoreboard.Team;
+import net.minecraft.text.LiteralText;
+import net.minecraft.util.Formatting;
+
+/**
+ * @author DarkKronicle
+ */
+
+public class BedwarsMod implements AbstractHypixelMod {
+
+	private final static Pattern[] GAME_START = {
+		Pattern.compile("^\\s*?Protect your bed and destroy the enemy beds\\.\\s*?$"),
+		Pattern.compile("^\\s*?Bed Wars Lucky Blocks\\s*?$"),
+		Pattern.compile("^\\s*?Bed Wars Swappage\\s*?$")
+	};
+
+	@Getter
+	private static BedwarsMod instance = new BedwarsMod();
+
+	@Getter
+	private final OptionCategory category = new OptionCategory("bedwars");
+
+	private final BooleanOption enabled = new BooleanOption("enabled", false);
+
+	public final BooleanOption hardcoreHearts = new BooleanOption(getTranslationKey("hardcoreHearts"), true);
+
+	public final BooleanOption showHunger = new BooleanOption(getTranslationKey("showHunger"), false);
+
+	public final BooleanOption displayArmor = new BooleanOption(getTranslationKey("displayArmor"), true);
+
+	public final BooleanOption bedwarsLevelHead = new BooleanOption(getTranslationKey("bedwarsLevelHead"), true);
+	public final EnumOption bedwarsLevelHeadMode = new EnumOption(getTranslationKey("bedwarsLevelHeadMode"),
+		BedwarsLevelHeadMode.values(),
+		BedwarsLevelHeadMode.GAME_KILLS_GAME_DEATHS.toString());
+
+
+	protected BedwarsGame currentGame = null;
+
+	@Getter
+	protected final TeamUpgradesOverlay upgradesOverlay;
+
+
+	protected final BooleanOption removeAnnoyingMessages = new BooleanOption(getTranslationKey("removeAnnoyingMessages"), true);
+
+
+	private final BooleanOption tabRenderLatencyIcon = new BooleanOption(getTranslationKey("tabRenderLatencyIcon"), false);
```

Comment:
It may be worthwhile to add to the description that in hypixel games everyone's ping is sent as 1